src/libraries/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -140,7 +140,7 @@ public void SendPingWithIPAddress(AddressFamily addressFamily)
             }
 
             SendBatchPing(
-                (ping) => ping.Send(localIpAddress),
+                (ping) => ping.Send(localIpAddress, 10000),
```

Comment:
```suggestion
                (ping) => ping.Send(localIpAddress, TestSettings.PingTimeout),
```
Like in other tests? The value is also 10000.